include/nigiri/rt/frun.h

```diff
@@ -39,6 +39,8 @@ struct frun : public run {
     clasz get_clasz(event_type = event_type::kDep) const noexcept;
     clasz get_scheduled_clasz(event_type = event_type::kDep) const noexcept;
 
+    route_color route_color(event_type = event_type::kDep) const noexcept;
```

Comment:
```suggestion
    route_color get_route_color(event_type = event_type::kDep) const noexcept;
```

Reply:
Our GCC version seems to not like that `struct route_color` and method `route_color` have the same name.